src/main/java/jto/processing/sketch/mapper/SketchMapper.java

```diff
@@ -287,13 +287,17 @@ public void mouseEvent(MouseEvent event) {
     }
 
     public int getMostRecentSurface() {
-    	return mostRecentSurface;
+        return mostRecentSurface;
     }
-    
+
     public void resetMostRecentSurface() {
-    	mostRecentSurface = 0;
+        mostRecentSurface = 0;
     }
-    
+
+    public boolean isMouseWithinActiveSurface() {
```

Comment:
I think this method name is a little misleading

`surfaceMapper.findActiveSurface` is also misleading haha. What that actually does is tell if you the current mouse position is inside _any_ of the surfaces.

```suggestion
    public boolean isMouseWithinAnySurface() {
```